app/models/student_profile.py

```diff
@@ -0,0 +1,29 @@
+from flask import current_app
+from flask_login import AnonymousUserMixin, UserMixin
+from itsdangerous import TimedJSONWebSignatureSerializer as Serializer
+from itsdangerous import BadSignature, SignatureExpired
+from werkzeug.security import check_password_hash, generate_password_hash
+
+from .. import db, login_manager
+
+class StudentProfile(db.Model):
+    __tablename__ = 'students'
+    id = db.Column(db.Integer, primary_key=True)
+    student_id = db.Column(db.Integer, db.ForeignKey('users.id'))
+    colleges = db.Column(db.String(10000), index = True) 
```

Comment:
Really small style point but in general for parameters, we don't include spaces between the `=` sign. So in this case, we would have `colleges = db.Column(db.Text, index=True)`